.github/workflows/android.yml

```
@@ -26,11 +26,15 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 42                                 # fetch last 42 commits to increase chances of determining "pretty" version based on previous known tag
```

```suggestion
          fetch-depth: 0                                  # fetch last 42 commits to increase chances of determining "pretty" version based on previous known tag
```
It's not like berty is a 1GB repo, I think it's worth downloading all the repo safe in case we merge more than 42 commits beetween the tag.

---

let's try

---

yep, I confirm, there is only about 1 more second added :+1: